### PR TITLE
add ability to pass data from environment variables to message headers

### PR DESCRIPTION
**Pull request by VitaliyKovalchuk**
add ability to pass data from environment variables to message headers using `ELASTICIO_ADDITIONAL_VARS_FOR_HEADERS` variable

### CODE REVIEW

**Comment by jhorbulyk** [comment]
Is there a corresponding GitHub Issue related to this PR?

**Comment by VitaliyKovalchuk** [comment]
Yes, this one [#2826](https://github.com/elasticio/elasticio/issues/2826)

File: lib/settings.js [hunk 1]
```diff
@@ -51,20 +53,32 @@ function readFrom(envVars) {
...
     };
 
+    if (envVars.ELASTICIO_ADDITIONAL_VARS_FOR_HEADERS) {
+        const vars = {};
+        envVars.ELASTICIO_ADDITIONAL_VARS_FOR_HEADERS
+            .split(',')
+            .map(env => env.trim())
+            .forEach(env => {
+                const key = env.indexOf(PREFIX) === 0 ? env.slice(PREFIX.length) : env;
+                vars[key] = envVars[env];
```
Comment (anton-kotenko) [inline lib/settings.js]: As far as i understand, at this moment envVars object contains key-value pairs
where key is prefixed. Here values are takes by key without prefix.
This seems to be strange. 
maybe tests works due to the fact that objects are passed by reference thought code?

Comment (VitaliyKovalchuk) [reply lib/settings.js]: discussed in PM

**Pull request merged**